examples/applications/plot_stock_market.py

```diff
@@ -149,15 +177,17 @@
 
 symbols, names = np.array(list(symbol_dict.items())).T
 
-quotes = [quotes_historical_yahoo_ochl(symbol, d1, d2, asobject=True)
-          for symbol in symbols]
+quotes = [
+    quotes_historical_google(symbol, d1, d2) for symbol in symbols
+]
 
-open = np.array([q.open for q in quotes]).astype(np.float)
-close = np.array([q.close for q in quotes]).astype(np.float)
+close = np.stack([q['close'] for q in quotes])
+open = np.stack([q['open'] for q in quotes])
 
 # The daily variations of the quotes are what carry most information
 variation = close - open
```

Comment:
Again, not this pr's fault, but open is a reserved keyword in python, could we rename these `close_quote` / `open_quote` or whatever the explicit finance terms are?